frontend/src/metabase/reference/segments/SegmentQuestions.jsx

```diff
@@ -83,22 +90,22 @@ class SegmentQuestions extends Component {
           error={loadingError}
         >
           {() =>
-            Object.keys(entities).length > 0 ? (
+            questions.length > 0 ? (
               <div className="wrapper wrapper--trim">
                 <List>
-                  {Object.values(entities).map(
-                    entity =>
-                      entity &&
-                      entity.id &&
-                      entity.name && (
+                  {questions.map(
+                    ({ _card: question }) =>
```

Comment:
Card is not exactly a Question, the rename is not entirely correct.

Reply:
I agree.
But the file is called SegmentQuestions. If I start renaming, I'll have to go all the way.

Reply:
Getting access to the underlying _card seems not great. Could you use Question methods instead?

Reply:
It was impossible only in two places.
1. `card.created_at` and
2. `card.creator.common_name`

I've used question methods for everything else.
https://github.com/metabase/metabase/pull/38500/commits/e79a1332fdeb37d939e39473d02dd61f451353ff